### PR TITLE
Add more content to middleware snippets.

### PR DESCRIPTION
**Pull request by iantalarico**
This is in an attempt to make it more explicit that the middleware needs to be first to be able to properly work.

cc: @SurferJeffAtGoogle

### CODE REVIEW

**Comment by jskeet** [comment]
Will wait until the errors have been fixed before reviewing :)

**Comment by iantalarico** [comment]
@jskeet Forgot to updated the csproj for all frameworks.  PTAL.

File: apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/AspNetCoreSnippets.cs [hunk 1]
```diff
@@ -49,6 +49,15 @@ public void Configure(IApplicationBuilder app)
         {
             // Use before handling any requests to ensure all unhandled exceptions are reported.
             app.UseGoogleExceptionLogging();
+
+            app.UseStaticFiles();
```
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/AspNetCoreSnippets.cs]: Is the order of these calls important? If so, let's call that out in comments.

Comment (iantalarico) [reply apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.Snippets/AspNetCoreSnippets.cs]: Kinda but not really unless you had some overlap in routes in static files and mvc.  This is just how they order them in templates.

**Pull request merged**